recipes/behave-html-formatter/recipe.yaml

```diff
@@ -0,0 +1,56 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  name: behave-html-formatter
+  version: "0.9.10"
+  python_min: '3.9'
+  python_max: '4.0'
```

Comment:
You only need to define `python_min` when it is higher than conda-forge's min. Also, we do not recommend the use of a `python_max`, it obscures the recipe and may prevent the proper update when we need to do so.

```suggestion
```